src/addAllAssetsToCompilation.js

```diff
@@ -31,7 +31,16 @@ function resolveOutput(compilation, addedFilename, outputPath) {
 async function addFileToAssets(
   compilation,
   htmlPluginData,
-  { filepath, typeOfAsset = 'js', includeSourcemap = true, hash = false, publicPath, outputPath, files = [] }
+  {
+    filepath,
+    typeOfAsset = 'js',
+    includeSourcemap = true,
+    includeGzipped = true,
```

Comment:
Having this default to true is a breaking change as the plugin will fail if you don't have the gz-file